pkg/finder/fsfinder.go

```diff
@@ -115,6 +172,17 @@ func (fsf FileSystemFinder) findOne(pathRoot string) ([]FileMetadata, error) {
 		depth = *fsf.Depth
 	}
 
+	// Error checking of config file option
+	if fsf.ConfigFile != "" {
+		if _, err := os.Stat(fsf.ConfigFile); os.IsNotExist(err) {
```

Comment:
I'm unsure what is expected here. I mean if there is an error different than NoExist. What do you expect?